frontend/src/js/components/settings/addonselection.js

```diff
@@ -11,77 +11,160 @@
-    [JSON.stringify(addons), JSON.stringify(features)]
+    [JSON.stringify(addons), JSON.stringify(features), action]
   );
-
   return (
     <>
+      {action && <ConfirmAddon name={action.name} onClose={() => setAction(null)} onConfirm={() => requestAddon()} variant={action.add ? 'add' : 'remove'} />}
       <h3 className="margin-top-large">Get more features with add-ons</h3>
 
       <div className="flexbox column">
         <p>
-          Extend Mender features with our add-ons. Select one or more from the list and submit to request add-ons to be added to your plan. We&apos;ll adjust
-          your subscription and confirm it with you.
+          Enhance your Mender plan with optional add-on packages – offering additional features to easily manage your software and devices over-the-air. Below
+          you can request which add-ons you’d like to be included with your plan.
         </p>
-        {relevantAddons.map(addOn => {
-          const isEligible = addOn.eligible.indexOf(updatedPlan) > -1;
-
-          return (
-            <div
-              key={addOn.name}
-              className={`planPanel ${classes.planPanel} addon ${isUpgrade ? 'upgrade' : ''} ${addOn.isEnabled ? 'active' : ''} ${isEligible ? '' : 'muted'}`}
-              onClick={e => (isEligible ? onAddOnClick(e, addOn.name, !addOn.isEnabled) : () => false)}
-            >
-              <Checkbox disabled={!isEligible} checked={addOn.isEnabled} />
-              <div className="bold">{addOn.title}</div>
-              {isUpgrade && (
-                <div className="flexbox column">
-                  <div className={`link-color bold ${classes.price}`}>{addOn[updatedPlan].price}</div>
-                  <div>{addOn[updatedPlan].deviceCount}</div>
-                </div>
-              )}
-              <InfoText variant="dense">{addOn.description}</InfoText>
-              {/* eslint-disable-next-line react/jsx-no-target-blank */}
-              <a className="margin-left-small" href="https://mender.io/plans/features" target="_blank" rel="noopener">
-                Learn more
-              </a>
-            </div>
-          );
-        })}
+        <div className="flexbox column">
+          {relevantAddons.map(addOn => {
+            const isEligible = addOn.eligible.indexOf(currentPlan) > -1;
+            return (
+              <div key={addOn.name} className={`${classes.planPanel} addon ${addOn.isEnabled ? 'active' : ''} ${isEligible ? '' : 'muted'}`}>
+                <a
+                  className={`${classes.link} flexbox center-aligned bold`}
+                  href={`https://mender.io/pricing/add-ons/${addOn.name}`}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  Mender {addOn.title}
+                  <LaunchIcon className="link-color margin-left-x-small" fontSize="small" />
+                </a>
+                {isUpgrade && currentPlanName !== 'Enterprise' ? (
```

Comment:
since we had the plan names change before this should likely check for the id:
```suggestion
                {isUpgrade && currentPlan.id !== PLANS.enterprise.id ? (
```